manifests/params.pp

```diff
@@ -63,7 +64,9 @@
       $lsb_major_release = regsubst($::lsbmajdistrelease, '^(\d+).*', '\1')
 
       if versioncmp($lsb_major_release, $ruby18_compare) > 0 {
-        $package_version = '1.9.1'
+        if versioncmp($lsb_major_release, $ruby23_compare) > 0 {
+          $package_version = '2.3'
+        } else { $package_version = '1.9.1'}
```

Comment:
Could you make the formatting consistent with the other blocks? So newline after `{` then the code then the `}` on another line.